sandboxed_containers/modules

```diff
@@ -0,0 +1 @@
+../modules
```

Comment:
Is this a symbolic link? I think is is but can't tell, so just checking with you :)